tests/performance/wp-content/mu-plugins/server-timing.php

```diff
@@ -75,6 +83,14 @@ static function () use ( $wpdb, $timestart ) {
 				$server_timing_values['db-queries']    = $wpdb->num_queries;
 				$server_timing_values['ext-obj-cache'] = wp_using_ext_object_cache() ? 1 : 0;
 
+				if ( function_exists( 'wp_cache_get_stats' ) ) {
+					$obj_cache_stats = array_values( wp_cache_get_stats() );
+
+					$server_timing_values['obj-cache-hits']   = $obj_cache_stats[0]['get_hits'];
+					$server_timing_values['obj-cache-misses'] = $obj_cache_stats[0]['get_misses'];
+					$server_timing_values['obj-cache-ratio']  = $obj_cache_stats[0]['get_hits'] / ( $obj_cache_stats[0]['get_hits'] + $obj_cache_stats[0]['get_misses'] );
+				}
```

Comment:
Since this specific list is duplicated across two places in the file and is now becoming larger, maybe add a quick function that gets these supplementary metrics that apply across the board? Then the two hook callbacks can reuse the function.

Reply:
Makes sense to deduplicate.

But I'll probably remove them anyway if I can't get correct numbers.